composer.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "ezsystems/cookbook-bundle",
+    "description": "eZ Platform Bundle with ready to use code samples",
+    "homepage": "https://github.com/ezsystems/CookbookBundle",
+    "license": "GPL-2.0",
+    "type": "ezplatform-bundle",
+    "authors": [
+        {
+            "name": "eZ dev-team & eZ Community",
+            "homepage": "https://github.com/ezsystems/CookbookBundle/contributors"
+        }
+    ],
+    "require": {
+        "php": "^5.6|^7.0",
+        "ezsystems/ezpublish-kernel": "^6.7"
```

Comment:
df67d3c04d76ef9c6ff114bb91d1d4fd8152d218 implies we at least need 5.4.5|6.0, but otherwise couldn't find anything.